src/table/components/withStyling.jsx

```diff
@@ -4,10 +4,9 @@ import PropTypes from 'prop-types';
 export default function withStyling(CellComponent) {
   const HOC = (props) => {
     const { styling, ...passThroughProps } = props;
+    const { selectedCellClass, ...style } = styling;
```

Comment:
moving the class name selectedCellClass out of styling to avoid using selectedCellClass: 'unselected', in CSS style